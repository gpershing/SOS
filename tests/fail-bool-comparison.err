Fatal error: exception Failure("Cannot equate bool and bool")
