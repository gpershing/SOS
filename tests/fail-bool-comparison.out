Cannot equate bool and bool
